theseus_playground/src/main.rs

```diff
@@ -0,0 +1,93 @@
+#![cfg_attr(
+    all(not(debug_assertions), target_os = "windows"),
+    windows_subsystem = "windows"
+)]
+
+use dunce::canonicalize;
+use std::{
+    collections::HashSet,
+    path::{Path, PathBuf},
+};
+use theseus::prelude::*;
+use tokio::sync::oneshot;
+
+// We use this function directly to call authentication procedure
+// Note: "let url = match url" logic is handled differently, so that if there is a rate limit in the other set causing that one to end early,
+// we can see the error message in this thread rather than a Recv error on 'rx' when the receiver is mysteriously droppped
+pub async fn authenticate_run() -> theseus::Result<Credentials> {
+    println!("Adding new user account to Theseus");
+    println!("A browser window will now open, follow the login flow there.");
+
+    let (tx, rx) = oneshot::channel::<url::Url>();
+    let flow = tokio::spawn(auth::authenticate(tx));
+
+    let url = rx.await;
+    let url = match url {
+        Ok(url) => url,
+        Err(e) => {
+            flow.await.unwrap()?;
+            return Err(e.into());
+        }
+    };
+
+    webbrowser::open(url.as_str())?;
+    let credentials = flow.await.unwrap()?;
+    State::sync().await?;
+    println!("Logged in user {}.", credentials.username);
+    Ok(credentials)
+}
+
+#[tokio::main]
+async fn main() -> theseus::Result<()> {
+    // Initialize state
+    let _ = State::get().await?;
+
+    // Clear profiles
+    let h = profile::list().await?;
+    for (path, _) in h.into_iter() {
+        profile::remove(&path).await?;
+    }
+
+    // Create vanilla minecraft instance
```

Comment:
I know this is the playground but the below could just use the profile create func right?